docs/pages/components/icon/examples/ExCustom.vue

```diff
@@ -0,0 +1,135 @@
+<template>
+    <section>
+        <div class="block">
+            <b-icon
+                pack="cap-icon"
+                icon="user"
+                size="is-small">
+            </b-icon>
+            <b-icon
+                pack="cap-icon"
+                icon="home"
+                size="is-small">
+            </b-icon>
+            <b-icon
+                pack="cap-icon"
+                icon="gear"
+                size="is-small">
+            </b-icon>
+        </div>
+
+        <div class="block">
+            <b-icon
+                pack="cap-icon"
+                icon="user">
+            </b-icon>
+            <b-icon
+                pack="cap-icon"
+                icon="home">
+            </b-icon>
+            <b-icon
+                pack="cap-icon"
+                icon="gear">
+            </b-icon>
+        </div>
+
+        <div class="block">
+            <b-icon
+                pack="cap-icon"
+                icon="user"
+                size="is-medium">
+            </b-icon>
+            <b-icon
+                pack="cap-icon"
+                icon="home"
+                size="is-medium">
+            </b-icon>
+            <b-icon
+                pack="cap-icon"
+                icon="gear"
+                size="is-medium">
+            </b-icon>
+        </div>
+
+        <div class="block">
+            <b-icon
+                pack="cap-icon"
+                icon="user"
+                size="is-large"
+                type="is-success">
+            </b-icon>
+            <b-icon
+                pack="cap-icon"
+                icon="home"
+                size="is-large"
+                type="is-info">
+            </b-icon>
+            <b-icon
+                pack="cap-icon"
+                icon="gear"
+                size="is-large"
+                type="is-primary">
+            </b-icon>
+        </div>
+
+        <button class="button is-dark">
+            <b-icon pack="cap-icon" icon="check"></b-icon>
+            <span>Finish</span>
+        </button>
+
+        <button class="button is-warning">
+            <b-icon pack="cap-icon" icon="check"></b-icon>
+            <span>Finish</span>
+        </button>
+
+        <button class="button is-warning">
+            <b-icon
+                pack="cap-icon"
+                icon="refresh">
+            </b-icon>
+            <span>Refresh</span>
+        </button>
+    </section>
+</template>
+
+<script>
+    import config, { setOptions } from "../../../../../src/utils/config"
+
+    const customIconConfig = {
+        customIconPacks: {
+            'cap-icon': {
+                sizes: {
+                    default: 'is-size-5',
+                    'is-small': '',
+                    'is-medium': 'is-size-3',
+                    'is-large': 'is-size-1'
+                },
+                iconPrefix: 'ci-',
+                internalIcons: {
+                    check: 'check',
```

Comment:
Only for consistency you might use always string as key, for example `'check': 'check'`